### PR TITLE
handle creation process of flanneldv1 resource

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/giantswarm/issues/1975.

### CODE REVIEW

File: service/resource/flanneldv1/create.go [hunk 1]
```diff
@@ -2,12 +2,61 @@ package flanneldv1
...
+		return microerror.Mask(err)
+	}
+
+	if daemonSetToCreate != nil {
+		r.logger.Log("cluster", keyv2.ClusterID(customObject), "debug", "creating the daemon set in the Kubernetes API")
+
+		namespace := keyv2.NetworkNamespace(customObject)
+		_, err = r.k8sClient.Extensions().DaemonSets(namespace).Create(daemonSetToCreate)
+		if apierrors.IsAlreadyExists(err) {
+			// fall through
```
Comment (JosephSalisbury) [inline service/resource/flanneldv1/create.go]: Would it be an error if the daemonset existed already? I don't see how that could happen.

Comment (xh3b4sd) [reply service/resource/flanneldv1/create.go]: Uncertain. I think this is just a safety net? We have this like in each resource.

Comment (JosephSalisbury) [reply service/resource/flanneldv1/create.go]: Good enough for me. Thought I'd bring it up.

Comment (xh3b4sd) [reply service/resource/flanneldv1/create.go]: Appreciated!

**Pull request merged**